### ISSUE TITLE
Story under user's perspective

### ISSUE THREAD

**Issue by wuminzhe** [darwinia-network/darwinia-msgport#237]
![image](https://github.com/darwinia-network/darwinia-msgport/assets/1608576/9e40248c-de47-4abd-a7f2-39a0f8c467e9)

**Comment by wuminzhe** [comment]
![image](https://github.com/darwinia-network/darwinia-msgport/assets/1608576/96b452df-0f0f-433d-a2a6-cdcb50ce3658)

**Comment by zzz6519003** [comment]
msgport gives me the feeling it's just 收钱做事(give me the money, I'll do the job)

**Comment by wuminzhe** [comment]
> msgport gives me the feeling that it's just 收钱做事(give me the money, I'll do the job)

yes, you are right. Does this way give you a more easy-understanding overview?

**Comment by wuminzhe** [comment]
![image](https://github.com/darwinia-network/darwinia-msgport/assets/1608576/c3f329ac-9b90-4889-96e0-92008cfe0937)
